.github/workflows/build-test-package.yml

```diff
@@ -41,7 +42,23 @@ jobs:
 
     - name: Get specific version of CMake, Ninja
       uses: lukka/get-cmake@v3.18.3
-
+    
+    - name: Download VTK
+      run: |
+        cd ..
+        git clone https://github.com/Kitware/VTK.git
+        cd VTK
+        git checkout ${{ matrix.vtk-git-tag }}
+    
+    - name: Build VTK
+      if: matrix.os != 'windows-2019'
+      run: |
+        cd ..
+        mkdir VTK-build
+        cd VTK-build
+        cmake -DCMAKE_C_COMPILER:FILEPATH="${{ matrix.c-compiler }}" -DCMAKE_CXX_COMPILER="${{ matrix.cxx-compiler }}" -DBUILD_SHARED_LIBS:BOOL=OFF -DCMAKE_BUILD_TYPE:STRING=RELEASE -DBUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF -DVTK_USE_SYSTEM_EXPAT:BOOL=ON -DVTK_USE_SYSTEM_ZLIB:BOOL=ON -DVTK_LEGACY_REMOVE=ON -DVTK_Group_Rendering=OFF -DVTK_Group_StandAlone=OFF -DVTK_RENDERING_BACKEND=None -DVTK_WRAP_PYTHON=OFF -DModule_vtkCommonCore=ON -DModule_vtkCommonDataModel=ON -DModule_vtkFiltersCore=ON -DModule_vtkImagingStencil=ON -DModule_vtkIOXML=ON -DModule_vtkIOImage=ON -DModule_vtkImagingSources=ON -DVTK_GROUP_ENABLE_Views=NO -DVTK_GROUP_ENABLE_Web=NO -DVTK_GROUP_ENABLE_Imaging=NO -DVTK_GROUP_ENABLE_Qt=DONT_WANT -DVTK_GROUP_ENABLE_Rendering=DONT_WANT -GNinja ../VTK
```

Comment:
We probably don't need `-DModule_vtkImagingStencil=ON -DModule_vtkIOXML=ON`, you could try removing this after you get the rest working.